From 030d433de30ccd6675e73d25f364ac9fff26b746 Mon Sep 17 00:00:00 2001
From: ni-ka <mail@nils-kaiser.de>
Date: Sun, 24 May 2020 17:37:59 +0000
Subject: [PATCH] Removed reference to gsconnect extension

---
 extension.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/extension.js b/extension.js
index d05b29f..776a5ed 100644
--- a/extension.js
+++ b/extension.js
@@ -64,11 +64,11 @@ updateSettings();
 let settingsChangedId = settings.connect('changed', updateSettings.bind());
 
 const Config = imports.misc.config;
-window.gsconnect = {
+window.wintile = {
 	extdatadir: imports.misc.extensionUtils.getCurrentExtension().path,
 	shell_version: parseInt(Config.PACKAGE_VERSION.split('.')[1], 10)
 };
-imports.searchPath.unshift(gsconnect.extdatadir);
+imports.searchPath.unshift(wintile.extdatadir);
 
 const KeyBindings = imports.keybindings
 let keyManager = null;
-- 
2.25.4

